docs/translations.md

```diff
@@ -0,0 +1,288 @@
+# ⚡ Multi languages support
+
```

Comment:
We need to add something about `react-i18next.d.ts`, it is pretty important.
Each time we add a new namespace we need to add it to this file otherwise Typescript will complain.
We can also build a script that generates the typing files automatically during `setupTranslation` (i don't know if this is easily doable)